src/main/java/at/ac/tuwien/kr/alpha/common/terms/Terms.java

```diff
@@ -0,0 +1,35 @@
+package at.ac.tuwien.kr.alpha.common.terms;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Convenience methods for {@link Term}s.
+ * 
+ * Copyright (c) 2020, the Alpha Team.
+ */
+public final class Terms {
```

Comment:
I suggest to rename that class, since it is only utility methods.

Reply:
Quoting Joshua Bloch, author of "Effective Java" and one of the JVM devs, [on utility classes](https://www.informit.com/articles/article.aspx?p=1216151&seqNum=4) :
> [...] They can also be used to group static methods, including factory methods (Item 1), for objects that implement a particular interface, in the manner of java.util.Collections. [...]
So for a utility class providing helpers around a specific type, it's actually pretty much a convention to use the type name in plural, and I personally think it makes for easily readable code.

Reply:
That is a good argument. :-)
Following the linked text, I'd also suggest to make the private constructor throw an exception and add a comment that this should not be instantiated.

Reply:
done